test/bdd/pkg/issuer/issuer_steps.go

```diff
@@ -250,12 +264,34 @@ func (e *Steps) retrieveProfile(id, name, issuerURL, supportedVCContexts,
 	return nil
 }
 
-func (e *Steps) walletConnect(issuerID string) error {
+func (e *Steps) walletConnect(profileID string) error {
 	state := uuid.New().String()
-	e.states[issuerID] = state
+	e.states[profileID] = state
+
+	resp, err := bddutil.HTTPDo(http.MethodGet, //nolint: bodyclose
+		fmt.Sprintf(AdapterURL+"/%s/connect/wallet?state=%s", profileID, state), "", "", nil,
+		e.bddContext.TLSConfig())
+	if err != nil {
+		return fmt.Errorf("failed to execute wallet request: %w", err)
+	}
+
+	defer bddutil.CloseResponseBody(resp.Body)
+
+	// validating only status code as the vue page needs javascript support
+	if resp.StatusCode != http.StatusOK {
+		// nolint:wrapcheck // ignore
```

Comment:
You should replace `// ignore` with a specific reason of why you need the `nolint`